puml/DetailedDesign-BeaconBeep.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
+start
+->Beacon beep
+Beep with beacon strength
+Beep tone 1-5 in Temp1
```

Comment:
I think that there is a version which will use the melody as beacon beep when setting it to some specific value, no? I am not sure with which version this is though, might be one of the unstable releases, but might be good to add this info maybe here, or in the page itself.

Reply:
beacon beep version is in repo branch https://github.com/mathiasvr/bluejay/tree/beacon-melody.
Current codebase is based on main Mathias's repo branch with edt patches.
I propose continuing documenting current codebase and after that adding beacon melody patches and documenting them.

Reply:
Makes sense.